### PR TITLE
Validates CSS files during CI

### PR DESCRIPTION
**Pull request by hao-fang**
I notice we didn't validate CSS files in CI, so this PR adds this check.

Also I did some refactor on the Makefile.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -25,3 +26,4 @@ format-check:
 	(isort -rc $(PYTHON_FILES) --check-only --multi-line=3 --trailing-comma --force-grid-wrap=0 --use-parentheses --line-width=88) && (black -t py37 --check $(PYTHON_FILES)) || (echo "run \"make format\" to format the code"; exit 1)
 	pylint -j0 $(PYTHON_FILES)
 	mypy --show-error-codes $(PYTHON_FILES)
+	prettier $(CSS_FILES) --check
```
Comment (hao-fang) [inline Makefile:29]: this checks the CSS files

File: static/css/main.css [hunk 2]
```diff
@@ -1,182 +1,182 @@
 body a {
-    color: #2294e0;
+  color: #2294e0;
```
Comment (hao-fang) [inline static/css/main.css:2]: this file is auto-formatted via `make format-web`

File: Makefile [hunk 3]
```diff
@@ -25,3 +26,4 @@ format-check:
 	(isort -rc $(PYTHON_FILES) --check-only --multi-line=3 --trailing-comma --force-grid-wrap=0 --use-parentheses --line-width=88) && (black -t py37 --check $(PYTHON_FILES)) || (echo "run \"make format\" to format the code"; exit 1)
 	pylint -j0 $(PYTHON_FILES)
 	mypy --show-error-codes $(PYTHON_FILES)
+	npx prettier $(CSS_FILES) --check
```
Comment (hao-fang) [inline Makefile:29]: this will check the CSS format
Also, I use `npx` to execute prettier so you don't need to add `./node_modules/.bin` to the `PATH`.

Comment (srush) [reply Makefile:29]: nice.

File: package-lock.json [hunk 4]
```diff
@@ -0,0 +1,14 @@
+{
```
Comment (srush) [inline package-lock.json:1]: Should this file be checked in?

File: static/css/main.css (hunk 2 above)
Comment (srush) [reply static/css/main.css:2]: Hmm, I am a little worried that prettier has different versions/config? It looks like this just changed whitespace?

Comment (srush) [reply static/css/main.css:2]: oh actually looks good. 2 is the default.

File: package-lock.json (hunk 4 above)
Comment (hao-fang) [reply package-lock.json]: oh yes.. this would make sure the CI environment installs the right packages via `npm ci`.

Comment (srush) [reply package-lock.json]: Ah gotcha. Makes sense!

**Pull request merged**